pkg/KubeArmorAnnotation/config/samples/policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-group-1-proc-path-block
```

Comment:
group-1?

Reply:
ksp-test1-proc-path-block

Reply:
left over files, removed the whole samples directory